browser/containers/Approvals.tsx

```diff
@@ -51,12 +51,16 @@ class Approvals extends React.Component<Props, State> {
     this.setState({
       approvalStatus: 'approved',
     });
+    (document.getElementById('approveBtn') as any).disabled = true;
```

Comment:
Don't modify the DOM in a React app, instead add something like `disabled={this.state.approvalStatus === 'approved'}` on the `<button>`